great_expectations/checkpoint/checkpoint.py

```diff
@@ -717,7 +717,11 @@ def instantiate_from_config_with_runtime_args(
         **runtime_kwargs,
     ) -> "Checkpoint":
         config: dict = checkpoint_config.to_json_dict()
-        config.update(runtime_kwargs)
+
+        for k, v in runtime_kwargs.items():
```

Comment:
@roblim Sorry, one minor thing -- please -- for style -- key: str and value: Any -- if not a problem (we now do type hints consistently -- at least trying to).  Thanks!

Reply:
Added in https://github.com/great-expectations/great_expectations/pull/4352/commits/074633aac5191357a6b4b17acdfb8c3762641d84